app/langswitch/LangMessages.scala

```diff
@@ -63,355 +71,659 @@ object LangMessages {
+    welsh   = "Mae’ch taliad TAW wedi dod i law."
+  )
 
-  val `Amount you claimed`: Message = Message("Amount you claimed")
+  val `Amount you claimed`: Message = Message(
+    english = "Amount you claimed",
+    welsh   = "Y swm a hawliwyd gennych"
+  )
 
-  val `Amount we'll pay you`: Message = Message("Amount we'll pay you")
+  val `Amount we'll pay you`: Message = Message(
+    english = "Amount we'll pay you",
+    welsh   = "Y swm y byddwch yn ei dalu i chi"
+  )
 
-  val `Amount to pay`: Message = Message("Amount to pay")
+  val `Amount to pay`: Message = Message(
+    english = "Amount to pay",
+    welsh   = "Y swm i’w dalu"
+  )
 
-  val `Amount we paid you`: Message = Message("Amount we paid you")
+  val `Amount we paid you`: Message = Message(
+    english = "Amount we paid you",
+    welsh   = "Y swm y gwnaethom ei dalu i chi")
 
-  val `Amount you paid`: Message = Message("Amount you paid")
+  val `Amount you paid`: Message = Message(
+    english = "Amount you paid",
+    welsh   = "Y swm a dalwyd gennych")
 
-  val `You need to make a VAT payment`: Message = Message("You need to make a VAT payment")
+  val `You need to make a VAT payment`: Message = Message(
+    english = "You need to make a VAT payment",
+    welsh   = "Mae’n rhaid i chi wneud taliad TAW"
+  )
 
-  val `You have no completed repayments`: Message = Message("You have no completed repayments")
+  val `You have no completed repayments`: Message = Message(
+    english = "You have no completed repayments",
+    welsh   = "Nid oes gennych ad-daliadau wedi’u gwneud"
+  )
 
-  val `No repayments in progress`: Message = Message("No repayments in progress")
+  val `No repayments in progress`: Message = Message(
+    english = "No repayments in progress",
+    welsh   = "Nid oes ad-daliadau ar y gweill")
 
-  val `no-in-progress-desc`: Message = Message("""If you are expecting a repayment and it is not showing here, check this page again in 24 hours.</p><p>If you submitted your return early check this page after your accounting period has ended.""")
+  val `no-in-progress-desc`: Message = Message(
+    """If you are expecting a repayment and it is not showing here, check this page again in 24 hours.</p><p>If you submitted your return early check this page after your accounting period has ended.""",
+    """Os ydych yn disgwyl ad-daliad ac nad yw’n ymddangos yma, gwiriwch y dudalen hon eto mewn 24 awr.</p><p>Os gwnaethoch gyflwyno’ch Ffurflen TAW yn gynnar, gwiriwch y dudalen hon ar ôl i’ch cyfnod cyfrifyddu wedi dod i ben."""
+  )
 
-  def `no-completed-desc`(url: String): Message = Message(s"""Completed repayments will only show here for 60 days after your repayment date. View your <a href="$url">VAT payments history.</a>""")
+  def `no-completed-desc`(url: String): Message = Message(
+    s"""Completed repayments will only show here for 60 days after your repayment date. View your <a href="$url">VAT payments history.</a>""",
```

Comment:
Nb: I've noticed here the `href` attribute value is quoted whereas in other messages it's left unquoted.

If the URL may contain funky characters, it may be worth quoting the `href` attribute value in the other messages.

Reply:
Sure, I'll change all messages.